pallets/capacity/src/migration/v3.rs

```diff
@@ -0,0 +1,159 @@
+use crate::{BalanceOf, BlockNumberFor, Config, FreezeReason, Pallet, StakingAccountLedger};
+use frame_support::{
+	pallet_prelude::{GetStorageVersion, IsType, Weight},
+	traits::{
+		fungible::MutateFreeze, Get, LockIdentifier, LockableCurrency, OnRuntimeUpgrade,
+		StorageVersion,
+	},
+};
+use parity_scale_codec::Encode;
+use sp_core::hexdisplay::HexDisplay;
+use sp_runtime::Saturating;
+
+const LOG_TARGET: &str = "runtime::capacity";
+
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+const STAKING_ID: LockIdentifier = *b"netstkng";
+
+/// The OnRuntimeUpgrade implementation for this storage migration
+pub struct MigrationToV3<T, OldCurrency>(sp_std::marker::PhantomData<(T, OldCurrency)>);
+impl<T, OldCurrency> MigrationToV3<T, OldCurrency>
+where
+	T: Config,
+	OldCurrency: 'static + LockableCurrency<T::AccountId, Moment = BlockNumberFor<T>>,
+	OldCurrency::Balance: IsType<BalanceOf<T>>,
+{
+	/// Translate capacity staked locked deposit to frozen deposit
+	pub fn translate_lock_to_freeze(account_id: T::AccountId, amount: OldCurrency::Balance) {
+		OldCurrency::remove_lock(STAKING_ID, &account_id); // 1r + 1w
```

Comment:
this seems more than 1r and 1r
can you please double-check this:
https://github.com/LibertyDSNP/frequency/pull/1779/files#diff-2fd628e078dc6cd044753cc133baa49c80e80c3f1a3d2cbe54396956179b5447R29